### ISSUE TITLE
[Accessibility]: No announcement or focus change on closing file view in ref panel

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#36625]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33510

### Problem description

When the Close file view button in the ref panel is clicked, there is no audio cue and the focus stays in place where the Close button was present.

### Expected behavior

- There should be some announcement that the file panel was closed.
- The focus should move to a sensible position, perhaps the line in the ref panel which triggered the opening of the file view (if that is available) or the file path.

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
